### DISCUSSION TITLE
lotus v1.6.0 and network v11 upgrade discussion

### DISCUSSION THREAD

**Discussion by jennijuju** [filecoin-project/lotus#5906] [General Q&A]
If you have any questions wrt the [v1.6.0 release and network v11 upgrade](https://github.com/filecoin-project/lotus/discussions/5905), leave a comment here!

**Comment by YiRuiLiher** [comment]
I need to verify my program, but there is no V1 sector in the calibration network to verify. Is there a better test method?
